src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx

```diff
@@ -93,14 +92,14 @@ export class CytoscapeReactWrapper extends React.Component<CytoscapeReactWrapper
     );
 
     this.cy = cytoscape(opts);
-    this.panzoom = this.cy.panzoom();
```

Comment:
@israel-hdez updated this code, it returns the `cy` instance and receives the init arguments or the `destroy` command:
see: https://github.com/cytoscape/cytoscape.js-panzoom/blob/master/cytoscape-panzoom.js#L561